conftest.py

```diff
@@ -277,7 +310,7 @@ def pytest_runtest_setup(item: Item) -> None:
         item.fixturenames.insert(0, "enabled_modelmesh_in_dsc")
 
     # The above fixtures require the global config to be updated before being called
-    item.fixturenames.insert(0, "updated_global_config")
+    updated_global_config(admin_client=get_client())
```

Comment:
you need to remove this.

Reply:
yep, this seems like a leftover

Reply:
I put the if to guard to no have tox to fail, but I kept the original flow that was calling the fixture, now the fixture is a normal function. But if you both think that is not needed anymore for `pytest_runtest_setup` I can remove it